### PR TITLE
Fix skipped tests

### LINKED ISSUE #864: Skipped tests
Fix skipped tests or remove if no longer relevant.

### PR DESCRIPTION
**Pull request by JCook21**
Closes #864 

- Fix test that is skipped
- Remove test that depends on ProcessMaker core
- Update test that depends on ProcessMaker core to run independently
- Add test case to assert that a text annotation can be added to a pool

### CODE REVIEW

File: src/components/inspectors/DateTimeExpression.vue [hunk 1]
```diff
@@ -8,6 +8,7 @@
       class="p-0"
       :value="convertFromUTC(value)"
       @input="emitValue"
+      data-test="date-picker"
```
Comment (JCook21) [inline src/components/inspectors/DateTimeExpression.vue]: This seemed to be the easiest solution to selecting this element in a test. Let me know if there's a better way.

File: tests/e2e/specs/Tasks.spec.js [hunk 2]
```diff
@@ -118,25 +118,4 @@ describe('Tasks', () => {
...
-    cy.contains('Task Assignment').next().select('User');
-    cy.contains('Assigned User')
-      .should('exist')
-      .next().find('.multiselect').should('exist');
-
-    cy.contains('Task Assignment').next().select('Group');
-    cy.contains('Assigned Group')
-      .should('exist')
-      .next().find('.multiselect').should('exist');
-  });
```
Comment (JCook21) [inline tests/e2e/specs/Tasks.spec.js]: This test seems to rely completely on ProcessMaker core so I thought deleting it was the best option.

File: tests/e2e/specs/TextAnnotation.spec.js [hunk 3]
```diff
@@ -44,18 +40,54 @@ describe('Text Annotation', () => {
...
+    <bpmn:association id="node_8" associationDirection="None" sourceRef="node_6" targetRef="node_7" />`;
+
+    cy.get('[data-test=downloadXMLBtn]').click();
+    cy.window()
+      .its('xml')
+      .then(xml => xml.trim())
+      .then(xml => {
+        expect(xml).to.contain(expectedXML.trim());
+      });
+  });
```
Comment (JCook21) [inline tests/e2e/specs/TextAnnotation.spec.js]: This change adds something we can assert about outside of core.

File: tests/e2e/specs/IntermediateCatchEvent.spec.js [hunk 4]
```diff
@@ -42,14 +42,11 @@ describe('Intermediate Catch Event', () => {
...
     typeIntoTextInput(nameInput, testString);
-
     cy.contains('Timing Control').click();
     cy.get('[data-test=intermediateTypeSelect]').select('Date/Time');
-    const startDateValue = '2019-02-27';
-    typeIntoTextInput('.start-date', startDateValue);
-
-    const startTimeValue = '00:30';
-    cy.get('[data-test=startTime]').select(startTimeValue, { force: true });
+    const startDateTime = '02/27/2019 12:30 AM';
```
Comment (lisa-fehr) [inline tests/e2e/specs/IntermediateCatchEvent.spec.js]: When I looked at this test, I tried to click the time in the new date/time picker. It just didn't seem easy to do. So I'm ok with this.

File: tests/e2e/specs/TextAnnotation.spec.js (hunk 3 above)
Comment (chipit24) [reply tests/e2e/specs/TextAnnotation.spec.js]: We have a function called `assertDownloadedXmlContainsExpected` in `utils.js`, could that be used here?

File: tests/e2e/specs/TextAnnotation.spec.js [hunk 5]
```diff
@@ -44,18 +40,54 @@ describe('Text Annotation', () => {
...
+    </bpmn:textAnnotation>
+    <bpmn:association id="node_4" associationDirection="None" sourceRef="node_3" targetRef="node_2" />
+  </bpmn:process>`;
+
+    cy.get('[data-test=downloadXMLBtn]').click();
+    cy.window()
+      .its('xml')
+      .then(xml => xml.trim())
+      .then(xml => {
+        expect(xml).to.contain(expectedXML.trim());
```
Comment (chipit24) [inline tests/e2e/specs/TextAnnotation.spec.js]: Same comment here regarding `assertDownloadedXmlContainsExpected`.

**Pull request merged**